.idea/misc.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I recommend that you add this files to your .gitignore as it is a personal configuration.

Reply:
Thx 👍 :D